x/coinswap/client/cli/query.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/coinswap/internal/types"
 )
 
+const (
+	nativeDenom = "atom"
```

Comment:
it is in the core code; its a parameter. The issue on the client is its useful to know what the native denom is but it is inefficient to do an extra query for it for every command especially since it doesn't change after genesis. One way of handling this is through a config file, which could be done at a later point